.gitignore

```diff
@@ -1,3 +1,7 @@
 env
 __pycache__
-*.jl
\ No newline at end of file
+*.jl
+.vscode/
+.idea/
```

Comment:
It can be helpful to share editor settings files with other contributors. For example the .vscode folder contains some settings that allow you to run the unit tests from a GUI in Visual Studio Code. You can use your discretion about including your .idea folder in the repository.

Reply:
That makes sense. I hadn't thought about that. I'll keep that in mind as I add debugging and testing settings.